data-non-standard.js

```diff
@@ -1218,7 +1212,7 @@ exports.tests = [
     firefox12: false,
     firefox16: true,
     firefox17: false,
-    firefox20: true,
+    firefox20: false,
     firefox28: false,
```

Comment:
> All browsers which have identical full field will nest data from previous marked version's value for each test and it's normal in this repo. Actually you loose data when you delete browser from browser list and not loose any useful data when you delete repeating values in tables (and especially you loose nothing when you delete data for browser not represented in browser list).
I agree with you, and I said it above.  I'll do a separate patch removing redundant entries like this, please help me merge it.

>There are no Firefox browsers with versions lower then 31 get rendered if you check obsolete browsers checkbox on. This is because of obsolete: very, fields in very old versions. There is nothing about very obsolete browsers in DOM.  So this is not really an optimisation.
I understand how this works, and I have marked some versions as "very obselete" myself.  I even have an open issue suggesting we mark more versions as "very obsolete".  But for non-standard features table, which is the one we are talking about here, no browsers are marked as "very obsolete".